syne_tune/tuner.py

```diff
@@ -76,6 +77,8 @@ def __init__(
         :param asynchronous_scheduling: whether to use asynchronous scheduling when scheduling new trials. If `True`,
         trials are scheduled as soon as a worker is available, if `False`, the tuner waits that all trials are finished
          before scheduling a new batch.
+        :param stop_immediately_when_stopping_criterion_met: how to deal with running trials when stopping criterion is
```

Comment:
The name is not ideal since we do not stop immediately when stopping condition is met (we do some clean-ups such as stopping running trials to avoid wasting ressource, save last result etc). Perhaps, `wait_trial_completion` or `wait_trial_completion_when_stopping`? (or do you have another one?)

Reply:
done